### PR TITLE
chore: upgrade @ant-design/tools

### PR DESCRIPTION
**Pull request by hengkx**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by hengkx** [comment]
等#27270 应该代码 风格 都可以了 我看看 怎么 把他 加入到 PR检测上个

**Comment by hengkx** [comment]
tools是升级了 依赖不兼容了吗？

**Comment by afc163** [comment]
我在 antd-tools 里升级了几个 loader，帮忙排查一下哪里出问题了需要修改。

**Comment by hengkx** [comment]
> 我在 antd-tools 里升级了几个 loader，帮忙排查一下哪里出问题了需要修改。

好的

**Comment by hengkx** [comment]
蛋疼 我本地 运行 正常

File: package.json [hunk 1]
```diff
@@ -155,7 +155,7 @@
   "devDependencies": {
     "@ant-design/bisheng-plugin": "^2.3.0",
     "@ant-design/hitu": "^0.0.0-alpha.13",
-    "@ant-design/tools": "^10.0.1",
+    "@ant-design/tools": "https://github.com/ant-design/antd-tools#fix-dist",
```
Comment (hengkx) [inline package.json:158]: 这个写法对吗？？？

**Comment by hengkx** [comment]
无语了 CI re run 是走的 缓存吗😂😂😂😂😂😂😂😂

**Comment by afc163** [comment]
看上去能过了。

**Pull request merged**